service/javadsl/server/src/main/java/com/lightbend/lagom/javadsl/server/ServiceGuiceSupport.java

```diff
@@ -136,7 +245,31 @@ default void bindServices(ServiceBinding<?>... serviceBindings) {
      * @param <T>              type constraint ensuring <code>serviceImplementation</code> implements <code>serviceInterface</code>
      */
     default <T extends Service> void bindService(Class<T> serviceInterface, T service) {
-        bindServices(serviceBinding(serviceInterface, service));
+        new LagomServiceBuilder(serviceBinding(serviceInterface, service), this).bind();
+    }
+
+    /**
+     * Binds a Service interface with an instance that implements it.
+     * <p>
+     * Inspects the service descriptor and creates routes to serve every call described.
+     * <p>
+     * Allows the configuration of additional Play routers.
+     * <p>
+     * Builds the {@link ServiceInfo} metadata.
+     *
+     * @param serviceInterface the interface class for a {@link Service}
+     * @param service          an instance of a class implementing <code>serviceInterface</code>
+     * @param additionalRouter      an additional Play Router
+     * @param additionalRouters     additional Play Routers (can be omitted)
+     * @param <T>              type constraint ensuring <code>serviceImplementation</code> implements <code>serviceInterface</code>
+     */
+    default <T extends Service> void bindService(Class<T> serviceInterface,
+                                                 T service,
+                                                 AdditionalRouter additionalRouter,
+                                                 AdditionalRouter... additionalRouters) {
+        new LagomServiceBuilder(serviceBinding(serviceInterface, service), this)
```

Comment:
Another reason is to avoid unnecessary allocation, but that's premature opt. This happens only on bootstrap so not code that gets executed over and over again. 

I will reconsider.

Reply:
I'm opting to keep the overload methods because. 
I don't want to exclude these methods in MiMa. I don't think it's a good idea to exclude them because we can accidentally break them in future releases without realising.

Let's keep MiMa happy and avoid trouble.

Reply:
I agree, better to keep compatibility here